servers/navigation/navigation_path_query_parameters_2d.cpp

```diff
@@ -102,6 +102,36 @@ real_t NavigationPathQueryParameters2D::get_simplify_epsilon() const {
 	return simplify_epsilon;
 }
 
+void NavigationPathQueryParameters2D::set_included_regions(const TypedArray<RID> &p_regions) {
+	_included_regions.clear();
+	for (const RID region_rid : p_regions) {
```

Comment:
You can resize instead of clearing, to make sure we only have at max one allocation.